src/garage/sampler/local_sampler.py

```diff
@@ -100,6 +116,38 @@ def obtain_samples(self, itr, num_samples, agent_update, env_update=None):
                 if completed_samples > num_samples:
```

Comment:
should it be `completed_samples >= num_samples`? Otherwise, there will always be one extra more iteration.

Reply:
Oh. Yeah, that's a bug. It's not a bug in this PR though, so I'll fix it in another PR.